arangograph/single-sign-on.md

```diff
@@ -25,17 +44,98 @@ can sync the information about user resources in real-time:
 manage the user resources.
 
-To learn how to generate a new API key, see the [API Keys](my-account.html#api-keys) section.
+To learn how to generate a new API key in the ArangoGraph Dashboard, see the
+[API Keys](my-account.html#api-keys) section.
 
 {% hint 'info' %}
 When creating an API key, it is required to select an organization from the
 list.
-{% endhint %}
\ No newline at end of file
+{% endhint %}
+
+## Configure SAML 2.0 and SCIM using Okta
+
+You can enable SSO for your ArangoGraph organization using Okta as an Identity
+Provider (IdP). For more information about Okta, please refer to the
+[Okta Documentation](https://help.okta.com/en-us/Content/index.htm?cshid=csh-index){:target="_blank"}.
+
+### Create the SAML app integration in Okta
+
+1. Go to Okta and select **Applications** from the left sidebar menu.
+2. Click **Create App Integration**.
+3. In the **Create a new app integration** dialog, select **SAML 2.0**.
+
+   ![ArangoGraph Create Okta App Integration](images/arangograph-okta-create-integration.png)
+4. In the **General Settings**, specify a name for your integration.
+
+   ![ArangoGraph Okta Integration Name](images/arangograph-okta-integration-name.png)
+5. Click **Next**.  
+6. Configure the SAML settings:
+   - For **Single sign-on URL**, use `https://auth.arangodb.com/login/callback?connection=ORG_ID`
+   - For **Audience URI (SP Entity ID)**, use `urn:auth0:arangodb:ORG_ID`
+   
+   {% hint 'info' %}
+   Replace **ORG_ID** with your organization identifier from the 
+   ArangoGraph Dashboard. To find your organization ID, go to the **User Toolbar**
+   in the top right corner, which is accessible from every view of the Dashboard,
+   and click **My organizations**.
+   {% endhint %}
+
+   ![ArangoGraph Okta SAML General Settings](images/arangograph-okta-saml-general-settings.png)
+7. In the **Attribute Statements** section, define the following custom attributes:
+   - email: `user.email`
+   - first_name: `user.firstName`
+   - last_name: `user.lastName`
+   - email_verified: `true`
+   - picture: `user.profileUrl`
+
+   ![ArangoGraph Okta SAML Attributes](images/arangograph-okta-saml-attributes.png)
+8. Click **Next**.
+9. In the **Configure feedback** section, select **I'm an Okta customer adding an internal app**.
+10. Click **Finish**. The SAML app integration is now created.
+
+### SAML Setup
+
+1. After the integration is created, the **SAML Signing Certificates** section 
+   is displayed under the **Sign On** tab.
+2. To finalize the SSO configuration, you must perform the SAML setup.
+3. Click **View SAML setup instructions**.
+
+   ![ArangoGraph Okta SAML Setup](images/arangograph-okta-saml-setup.png)
+4. The setup instructions include the following items:
+   - Identity Provider Single Sign-On URL
+   - Identity Provider Issuer
+   - X.509 Certificate
+5. Copy the IdP settings, download the certificate, and share them with the
```

Comment:
So two things here:
1. We should be prescriptive on exactly what they should copy (is there a "copy IdP settings" button)? 
2.  We should give some advice on how they can share these with us. There are two paths as I see it, one where they're already engaged with us, and a path where they're trying to self-service. Perhaps we can just say "share them with the ArangoGraph team via an ArangoGraph Support ticket".

Reply:
And it would be good to instruct users not to send anything over the wire unencrypted, e.g. via normal email.

Reply:
1. There is no copy button for IdP settings (you need to manually copy both fields). However, there is a "Download X.509 certificate" button. 
2. We can instruct them to always share these via an ArangoGraph Support ticket.